flocker/dockerplugin/test/test_api.py

```diff
@@ -174,6 +174,40 @@ def test_path(self):
                            u"Mountpoint": u"/flocker/{}".format(dataset_id)}))
         return d
 
+    def test_unknown_path(self):
+        """
+        ``/VolumeDriver.Path`` returns an error when asked for the mount path
+        of a non-existent volume.
```

Comment:
:question: When might this occur? Maybe when `docker inspect`  is pointed at a flocker backed container that had been stopped and when the Flocker volume had been deleted?